src/java_bytecode/java_bytecode_language.h

```diff
@@ -177,6 +178,9 @@ class java_bytecode_languaget:public languaget
   java_string_library_preprocesst string_preprocess;
   std::string java_cp_include_files;
 
+  // list of classes to force load even without reference from the entry point
+  std::vector<irep_idt> java_load_classes;
```

Comment:
this fields seems duplicated here and in `java_class_loader.h`, if they are different I suggest finding better names